### PR TITLE
#Feature - Create pages to view orders

### PR DESCRIPTION
**Pull request by omobosteven**
#### Description of Task to be completed?
- Create pages to view orders for the day
- Add a new line to the end of every file

#### How should this be manually tested?
- Navigate to https://omobosteven.github.io/book-a-meal/user-order.html
- Navigate to https://omobosteven.github.io/book-a-meal/orders.html

### CODE REVIEW

File: UI/orders.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  </div>
+
+
+<footer class="footer">
+  <p>copyright<span>&copy;</span> 2018</p>
+</footer>
+
+  <script src="js/main.js"></script>
+</body>
+</html>
```
Comment (ayodelevm) [inline UI/orders.html]: This red marker means you didn't end this file with a new line. You should always end your file with a new line. Please fix that, and without add a new commit

File: UI/user-order.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    </div>
+  </div>
+
+  <footer class="footer">
+    <p>copyright<span>&copy;</span> 2018</p>
+  </footer>
+
+  <script src="js/main.js"></script>
+</body>
+</html>
```
Comment (ayodelevm) [inline UI/user-order.html]: The same goes for here

File: UI/orders.html (hunk 1 above)
Comment (omobosteven) [reply UI/orders.html]: I have fixed it

File: UI/user-order.html (hunk 2 above)
Comment (omobosteven) [reply UI/user-order.html]: I have fixed this too and every file

**Pull request merged**